backend.ps1

```diff
@@ -800,23 +844,34 @@ class HypervBackend : Backend {
 
     static [HypervBackend] Deserialize([string] $Json) {
         $deserialized = ConvertFrom-Json -InputObject $Json
-        $HypervBackend = [HypervBackend]::new(@($deserialized.ComputerName))
+        $h = @{}
+        $deserialized.psobject.properties | foreach {$h[$_.Name] = $_.Value}
+        $HypervBackend = [HypervBackend]::new($h)
         return $HypervBackend
     }
 
-    [void] CleanupInstance ($InstanceName) {
-        write-verbose ("Cleaning $InstanceName on backend " + $this.Name) 
+    [String] GetPathFromInstanceName ($InstanceName) {
+        $Path = $this.WorkingVHDsPath + "/$InstanceName" + $this.Suffix + ".vhdx"
```

Comment:
ditto - see the string formatting
$this.WorkingVHDsPath + "/$InstanceName" + $this.Suffix + ".vhdx" ->
{0}/{1}{2}.vhdx" -f @(....)